src/cascadia/TerminalCore/Terminal.cpp

```diff
@@ -989,6 +989,7 @@ void Terminal::_WriteBuffer(const std::wstring_view& stringView)
     }
 
     cursor.EndDeferDrawing();
+    _NotifyTerminalCursorPositionChanged();

```

Comment:
If we do it like that, I believe it's easier to understand if you remove the call to `_NotifyTerminalCursorPositionChanged` from `_AdjustCursorPosition` entirely and instead add it at the end of `CursorLineFeed`.

That way both functions have the exact same "code structure" of calling `_NotifyTerminalCursorPositionChanged` at the end. This would feel more consistent and logical to me.

Reply:
Alternatively, we might be able to get away with once _per `Write`_...